dots-data.Rmd

```diff
@@ -107,7 +107,23 @@ If you want your vector to have names, the problem is harder, and there's relati
 
 ## Selecting variables
 
-A number of funtions in the tidyverse use `...` for selecting variables. For example, `tidyr::fill()` lets you fill in missing values based on the previous row:
+A number of funtions in the tidyverse, like dplyr's scoped verbs and `ggplot2::facet_grid()`, require the user to explicitly quote the input. I now believe that this is a suboptimal interface because it is more typing (`var()` is longer than `c()`, and you must quote even single variables), and arguments that require their inputs to be explicitly quoted are rare in the tidyverse. 
+
+```{r, eval = FALSE}
+# existing interface
+dplyr::mutate_at(mtcars, vars(cyl:vs), mean)
+# what I would create today
+dplyr::mutate_at(mtcars, c(cyl:vs), mean)
+
+# existing interface
+ggplot2::facet_grid(rows = vars(drv), cols = vars(vs, am))
+# what I would create today
+ggplot2::facet_grid(rows = drv, cols = c(vs, am))
+```
+
+The `vars()` function is a simple wrapper around `quos()`, and adds no value compared to using `c()` directly, which requires quoting by the called function.
```

Comment:
I wouldn't say it adds no value. It can be useful to take selections in standard eval functions.